useful_resources/sample_code/ngpvan_sample_list.py

```diff
@@ -0,0 +1,73 @@
+# ### METADATA
+
+# Connectors: VAN
+# Description: Creates a new saved list from a random sample of an existing saved list in VAN
+
+# ### CONFIGURATION
+
+# Set the configuration variables below or set environmental variables of the same name and leave
+# these with empty strings.  We recommend using environmental variables if possible.
+
+config_vars = {
+    # VAN
+    "VAN_API_KEY": "",
+    "VAN_DB_NAME": "",  # One of MyVoters, MyCampaign, MyMembers, EveryAction
+    "VAN_FOLDER_ID": "",  # Can be found in the URL on a Folder page in the FolderID param
+    "VAN_SAVED_LIST_ID": "",
+    "VAN_SAMPLE_LIST_SIZE": "",  # Must be a string
+    "VAN_SAMPLE_LIST_NAME": "",
+    # AWS
+    "AWS_ACCESS_KEY_ID": "",
+    "AWS_SECRET_ACCESS_KEY": "",
+    "S3_BUCKET": "",  # Bucket that the sample list data should be uploaded to
+}
+
+
+# ### CODE
+
+from parsons import VAN  # noqa: E402
+from parsons import logger  # noqa: E402
+import os  # noqa: E402
+import random  # noqa: E402
```

Comment:
I've been told that convention is to put all the simple `import`s at the top, and more complicated `from`s at the bottom. That's a bit more nitpicky than I like to be (not to mention I don't even fully understand the reasoning behind it), but as a sample script, I figure we should model the convention.

Reply:
Agreed with you here as well as the above point, but I was going off of another script `apply_activist_code.py` so not sure if this is something we should apply across all scripts first?

Reply:
Updated for now because it's inconsistent across scripts!